examples/plot_simulate_evoked.py

```diff
@@ -36,10 +41,51 @@
 
 ###############################################################################
 # Let us first create our network from the params file and visualize the cells
-# inside it.
-net = Network(params)
+# inside it. The default behaviour of Network is to add and instantiate six
+# 'default' drives, but we will suppress that by setting the
+# `initialise_hnn_drives`-argument to `False`.
+net = Network(params, initialise_hnn_drives=False)
 net.plot_cells()
 
+###############################################################################
+# The network of cells is now defined, to which we add external drives as
+# required. Weights are prescribed separately for AMPA and NMDA receptors
+# (receptors that are not used can be omitted or set to zero)
+
+# Distal evoked drive
+weights_ampa_d1 = {'L2_basket': 0.006562, 'L2_pyramidal': .000007,
+                   'L5_pyramidal': 0.142300}
+weights_nmda_d1 = {'L2_basket': 0.019482, 'L2_pyramidal': 0.004317,
+                   'L5_pyramidal': 0.080074}
+dispersion_time_d1 = {'L2_basket': 0.1, 'L2_pyramidal': 0.1,
+                      'L5_pyramidal': 0.1}
+net.add_evoked_drive(
+    'evdist1', mu=63.53, sigma=3.85, numspikes=1, weights_ampa=weights_ampa_d1,
+    weights_nmda=weights_nmda_d1, location='distal', seedcore=2,
+    space_constant=3., dispersion_time=dispersion_time_d1)
+
+# First proximal evoked drive
+weights_ampa_p1 = {'L2_basket': 0.08831, 'L2_pyramidal': 0.01525,
+                   'L5_basket': 0.19934, 'L5_pyramidal': 0.00865}
+dispersion_time_prox = {'L2_basket': 0.1, 'L2_pyramidal': 0.1,
+                        'L5_basket': 1., 'L5_pyramidal': 1.}
+# all NMDA weights are zero; pass None
+net.add_evoked_drive(
+    'evprox1', mu=26.61, sigma=2.47, numspikes=1, weights_ampa=weights_ampa_p1,
+    weights_nmda=None, location='proximal', seedcore=2,
+    space_constant=3., dispersion_time=dispersion_time_prox)
+
+# Second proximal evoked drive. NB: only AMPA weights differ from first
+weights_ampa_p2 = {'L2_basket': 0.000003, 'L2_pyramidal': 1.438840,
+                   'L5_basket': 0.008958, 'L5_pyramidal': 0.684013}
+# all NMDA weights are zero; omit weights_nmda (defaults to None)
+net.add_evoked_drive(
+    'evprox2', mu=137.12, sigma=8.33, numspikes=1,
+    weights_ampa=weights_ampa_p2, location='proximal',
+    seedcore=2, space_constant=3., dispersion_time=dispersion_time_prox)
+
+# verify that three drives added
+print(net.external_drives)
```

Comment:
the output doesn't look very meaningful to me ...

Reply:
Me neither. The new `NetworkDrive` class has a `repr`, but `external_drives` is just a plain dict of those. So I guess the question is: what do your want `NetworkDrive.__repr__` to contain?